source/lib.civet

```diff
@@ -2347,19 +2341,19 @@ function processDeclarationConditionStatement(s: IfStatement | IterationStatemen
         expression: ref
         parent: s
       s.children[1] = s.condition
+      updateParentPointers s
```

Comment:
Does this need a `parent` passed in?

Reply:
I don't think so. `updateParentPointers(x)` updates `x.children` to point to `x`. The second argument is if you also want to set `x`'s parent, but it's optional. `s` is at the top level of what this function considers, so we don't know anything special about its parent.

Reply:
Ok, upon re-reading the method I see that you are correct, thanks!